app/lang/cy.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Yeah, I prefer this too, otherwise you end up typing out the English version twice. I think it would only make sense to do it that way if we had an automated way of extracting the strings to translate.

Reply:
the downside to this way is that you need to come up with unique keys for everything that make some kind of sense, but I'm hoping that shouldn't be too hard

Reply:
I prefer this approach too - a similar approach is used in DD for Symfony translations. Having a key with some context in makes the templates easier to parse and update quickly.